sql/2020/20_Caching/appcache_and_serviceworkers.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+# Use of AppCache and ServiceWorkers
+SELECT
+  SUBSTR(url, 0, 5) http_type,
+  JSON_EXTRACT_SCALAR(report, "$.audits.appcache-manifest.score") AS using_appcache,
+  JSON_EXTRACT_SCALAR(report, "$.audits.service-worker.score") AS using_serviceworkers,
+  count(*) AS occurences     

```

Comment:
Calculate the relative % since the absolute number of occurrences may be hard for readers to put in context.

```suggestion
  COUNT(0) AS occurrences,
  SUM(COUNT(0)) OVER () AS total,
  COUNT(0) / SUM(COUNT(0)) OVER () AS pct
```